.gitlab-ci.yml

```diff
@@ -775,12 +775,14 @@ test_accep_raspberrypi4:
     reports:
       junit: results_accep_raspberrypi4.xml
 
-test_backend_integration:
+test_backend_integration_enterprise:
   only:
     variables:
       - $RUN_INTEGRATION_TESTS == "true"
   stage: test
   image: docker/compose:alpine-1.27.4
+  variables:
+    TESTS_TYPE: "enterprise"
```

Comment:
Observation:
`TESTS_TYPE` sounds a little odd to me, is `TEST_TYPE` better?

Reply:
I would suggest `TEST_SUITE`.